### DISCUSSION TITLE
tmux Support

### DISCUSSION THREAD

**Discussion by zwhitchcox** [neovide/neovide#1516] [Ideas]
I'm a fan of using tmux with vim, and the combination is extremely powerful. Just wondering if you have considered building an animated tmux client too, as I think this would be very cool!

**Comment by semi710** [comment]
yeah, i also want that neovide can open tmux and then neovim in it.
something like `neovim --tmux "session-name" args` so that i can access my tmux floating terminal in neovide and all tmux stuff........

**Comment by jla2000** [comment]
I think something like this could be achieved through some neovim plugin (eg. this one https://github.com/hkupty/nvimux)
But ofc having there a real tmux instance running inside would be very cool.

**Comment by dan-myles** [comment]
God this would be a god send. An animated sleek terminal multiplexer? Sign me up!
